lib/rubocop/cop/style/method_call_with_args_parentheses.rb

```diff
@@ -371,6 +373,10 @@ def regexp_slash_literal?(node)
           node.regexp_type? && node.loc.begin.source == '/'
         end
 
+        def numeric_with_operator?(node)
```

Comment:
I think this should be a node extension.

Reply:
I have extracted `NumericNodes#sign?` are you okay with it?